src/System.Reflection.Metadata/src/Configurations.props

```diff
@@ -8,7 +8,10 @@
     <BuildConfigurations>
       $(PackageConfigurations);
       netcoreapp;
+      netcoreappaot-Windows_NT;
```

Comment:
Why do we need these addition configs? Is the aot build actually different from the non-aot build?

Reply:
I thought this is the thing that controls UAP won't get the netstandard version of the assembly from the NuGet, but I won't pretend I know what this file is doing. I will revert this.

Reply:
uap config is used for uapaot as well by default and takes precedence over nestandard. You need both uap and uapaot only if they are different.

Same for netcoreapp.